src/cli/abstraction/mod.rs

```diff
@@ -4,12 +4,12 @@ use clap::Args;
 pub(crate) struct AuthArguments {
 	#[arg(help = "Database authentication username to use when connecting")]
 	#[arg(env = "SURREAL_USER", short = 'u', long = "username", visible_alias = "user")]
-	#[arg(default_value = "root")]
-	pub(crate) username: String,
+	#[arg(default_value = None)]
```

Comment:
I doubt we need to explicitly say the default is `None` for `Option`'s

Reply:
good point! changed